lib.rs

```diff
@@ -301,7 +370,7 @@ impl<A: Array> Drop for SmallVecData<A> {
 /// assert!(v.spilled());
 /// ```
 pub struct SmallVec<A: Array> {
-    len: usize,
+    capacity: usize,
```

Comment:
Using `capacity` here is a little confusing to me, but I'm not sure what a better name would be.  Maybe something like `len_cap`?  Maybe `capacity` is okay, but there should be a comment here describing the semantics.

Reply:
Good point.